libs/sdk-backend-tiger/src/backend/index.ts

```diff
@@ -389,3 +398,41 @@ function createHeaders(implConfig: TigerBackendConfig, telemetry: TelemetryData)
 function isNotAuthenticatedResponse(err: any): boolean {
     return err?.response?.status === 401;
 }
+
+function retryMechanism(client: AxiosInstance): AxiosInstance {
+    // could use axios-retry https://www.npmjs.com/package/axios-retry
+    axiosRetry(client, {
+        retries: 3,
+        retryDelay: (retryCount) => retryCount * 1000,
+        retryCondition: (error) => {
+            return error?.response?.status === 400;
+        },
+    });
+
+    // interceptors
+    client.interceptors.response.use(identity, (error) => {
+        const response: AxiosResponse = error.response;
+        const config = error.config;
+
+        if (response.status === 429 && config.retry < TOTAL_NUMBER_OF_RETRIES) {
+            config.retry = config.retry + 1;
```

Comment:
How it will work when everything gets back to normal? Is there a reset for these props?